modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -621,6 +621,83 @@ definition {
 		task ("Then I can see the message 'You can submit up to 2 claim(s)'") {
 			PRMMDFRequest.assertClaimMessage(key_claimMessage = "You can submit up to 2 claim(s).");
 		}
+	
+	}
+	
+	@description = "This is a test for LPS-174679.Verify that is possible to export filtered MDF’s"
+    @priority = 4
+    test CanExportFilteredMDFs {
+		
+		task("When another account is created"){
+		JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Company Name 2",
+				accountEntryType = "Business");
+
+			JSONAccountEntryUser.addExistUsertoAccount(
+				accountEntryName = "Company Name 2",
+				userEmailAddress = "partner@liferay.com");
+		
+		}
+
+		task("Then I will add two orders of MDF"){
+		for (var n : list "1,2") {
+			PRMNavigator.goToNavItem(navItem = "Marketing");
+
+			PRMMarketing.gotoAddNewMDF();
+
+			PRMMDFRequest.createNewRequest(
+				accountName = "Company Name ${n}",
+				activityDescription = "Activity Description",
+				activityName = "Activity Name ${n}",
+				additionalOptions = "6.x to DXP Migration",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name ${n}",
+				countryName = "Global",
+				followUpStrategy = "test",
+				goalsOptions = "Lead generation",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetMarketOptions = "Education,Energy",
+				targetOfLeads = "test",
+				targetRolesOptions = "Administrator",
+				typeOfActivity = "Miscellaneous Marketing");
+			}
+		
+		}
+			
+			task("When the created MDFs are filtered"){
+			Click(
+           		 key_filterOption = "Filter",
+           		 locator1 = "AppBuilder#CHOOSE_SEARCH_FILTER_BY_OPTION");
+
+										
+			Click(locator1 = "PRMMDFRequest#PARTNER_BUTTON");
+    	
+					
+		for (var box : list "1,2") {
+			Click.clickAtNotVisible(
+				checkboxName = "Company Name ${box}",
+				locator1 = "PRMMDFRequest#CHECKBOX_ACCOUNT");
+			}
+
+			Click(locator1 = "PRMMDFRequest#APPLY_BUTTON");
```

Comment:
Hi @juliosilveira77, could you please create a macro to filter the MDF's? Thanks!